src/service/http/fetcher.ts

```diff
@@ -431,3 +432,31 @@ export const fetchMaintenanceInfo = () =>
   axiosIns(`/statistics/maintenance_info`).then((res: AxiosResponse) =>
     toCamelcase<Response.Wrapper<State.MaintenanceInfo>>(res.data.data),
   )
+
+export const exportTransactions = ({
+  type,
+  id,
+  startDate,
+  endDate,
+  fromHeight,
+  toHeight,
+  tab,
+}: {
+  type: State.TransactionCsvExportType
+  id?: string
+  startDate?: Dayjs
+  endDate?: Dayjs
+  fromHeight?: number
+  toHeight?: number
+  tab: string
+}) =>
+  axiosIns
+    .get(`http://3.112.30.76:8183/api/v1/${type}/download_csv${id ? `/${id}` : ''}`, {
```

Comment:
The URL is incorrect and should not be exposed in codebase

Reply:
It seems that the API is incorrect

From the code we can get API: https://ckb-explorer-api.staging.layerview.io/api/v1/address_transactions/download_csv/ckt1qzda0cr08m85hc8jlnfp3zer7xulejywt49kt2rr0vthywaa50xwsqt2yx7pkuk3ue203ck76sqvl7jxqa2ff3sn43zg5?start_date=2023-05-24&end_date=2023-05-26

which should be 

https://ckb-explorer-api.staging.layerview.io/api/v1/address_transactions/download_csv?id=ckt1qzda0cr08m85hc8jlnfp3zer7xulejywt49kt2rr0vthywaa50xwsqt2yx7pkuk3ue203ck76sqvl7jxqa2ff3sn43zg5&start_date=2023-05-24&end_date=2023-05-26

Reply:
notice that `tx hash` should be passed as `id` in query parameters